.env-example

```diff
@@ -0,0 +1,23 @@
+# Required redis host for backing the queues for Machinery
+MACHINERY_REDIS_HOST=redis://localhost:6379
```

Comment:
comment - I'd rather pick a proper message broker at the beginning if we test the machinery. e.g. rabbitMQ
not sure how machinery handles the message delivery, but a message broker sounds more reliable for message delivery/retry to me.

Reply:
No message brokers (including rabbit, redis and kafka) have retry functionality.  We need a PVC to ensure we store/save all messages for processing, so IMO, up to ops on comfort for ensuring we have a resilient service to back this. Can by anything that we are comfortable using.

Reply:
yeah I mean rabbitMQ is message broker but Redis is just a memory cache that has a queue. So would like the whole task queue backed by a message broker, which sounds more reliable to me. Can discuss in PR meeting